### PR TITLE
Blog : Added intro post for GSoC 2017

### PR DESCRIPTION
**Pull request by digib0y**
- Added blog post for GSoC 2017
- Renamed old blog from Deepanshu to digib0y
- Added redirect from old post to new post

### CODE REVIEW

**Comment by digib0y** [comment]
Please do not merge, I found few errors in the blog post.

**Comment by digib0y** [comment]
Good to go. Thanks.

File: content/blog/digib0y/2017-05-06_[gsoc_2017]_adding_harfbuzz_support_to_haiku.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+>Third Month of Coding(August 1 - August 30)
+	-Test for already supported scripts(latin,CKJ) and making sure that integration doesnt break it.
+	-Creating Test Suites with Hindi Samples.
+	-Integration of Harfbuzz in test_app_server rendering infrastructure and the necessary changes in libbe.
+
+Apart from this I will complete translation of Haiku to Hindi.
+
+**References:**
+-[PDF copy of my GSoC Proposal](https://www.google.com)
```
Comment (pulkomandy) [inline content/blog/digib0y/2017-05-06_[gsoc_2017]_adding_harfbuzz_support_to_haiku.md]: This link is not correct.

File: content/blog/digib0y/2017-05-06_[gsoc_2017]_adding_harfbuzz_support_to_haiku.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+type = "blog"
+title = "[GSoc 2017] Adding Harfbuzz support to Haiku"
+author = "digib0y"
+date = "2017-05-06 22:36:57+00:00"
+tags = ["haiku", "software", "GSoC2017", "students", "summerofcode", "google", "GSoC", "Google Summer of Code"] 
++++
+I am Deepanshu(Trac: digib0y IRC:digib0y), I am one of the seven students selected for GSoC this year from Haiku. I will be working to add Harfbuzz support this summer.
+
+**Quick intro:**
+I am an engineering student of Christ University, India. I'am pursuing a degree in Bachelor of Technology in Computer Science and Engineering.
```
Comment (pulkomandy) [inline content/blog/digib0y/2017-05-06_[gsoc_2017]_adding_harfbuzz_support_to_haiku.md]: You have a strange character here. Should be just a space?

File: content/blog/digib0y/2017-05-06_[gsoc_2017]_adding_harfbuzz_support_to_haiku.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+I have been a previous Google Code-In student where I worked with Haiku for the first time, I can recall the first time I ever connected to an IRC channel during GCI 2014; it was both my interest in the GCI task and my attraction to the positive, friendly environment on #haiku that convinced me to continue working with Haiku.
+
+**Project Description:**
+Text is the primary means of communication in computers, and is bound to be so for the decades to come. With the widespread adoption of Unicode as the canonical character set for representing text a whole new domain has been opened up in a desktop system software design.
+
+HarfBuzz uses GLib for Unicode functions (to get few Unicode character properties an normalisation). It can also use ICU for this, or a small bundled library called UCDN. If HarfBuzz is built without GLib or ICU, it will use UCDN. Alternatively we can provide the Unicode callbacks to interface with something else. However, we will be using ICU which is in the core of Haiku.
```
Comment (pulkomandy) [inline content/blog/digib0y/2017-05-06_[gsoc_2017]_adding_harfbuzz_support_to_haiku.md]: This jumps a bit fast into the technical details. The website is also for our end-users, so maybe you could state what the problem is. Something like this:

Haiku got localization support in 2009, however, it only works for languages with layouting rules similar to the latin alphabet. Other scripts such as arabic, devanagari, or CJK, have several problems ranging from minor (word breaks at unexpected places) to unreadable text (characters rendered in the wrong direction, or not rendered at all).

The goal of this project is to improve the situation, with a focus on Devanagari which is a script shared by several languages used in India.

**Pull request merged**